xml/System.Collections.Generic/IComparer`1.xml

```diff
@@ -67,12 +67,12 @@
   </Docs>
   <Members>
     <Member MemberName="Compare">
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; int Compare(T x, T y);" FrameworkAlternate="netframework-2.0;netframework-3.0;netframework-3.5" />
```

Comment:
will this be fixed now or will I see this jumping from time to time?

Reply:
So to answer your first question, yes, this will be fixed from here on out ... the way this new version of mdoc works is that it will recreate every signature, every time. That means that they will always be in the same order, with no possibility of them jumping around as they did in the past.

For the second question it's a bit more complicated. The c++/cli (and possibly the other c++ formatters? didn't check) doesn't support parameter variance and contravariance. As you can see in [this doc](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/covariance-contravariance/):

> In **.NET Framework 4 or newer** C# supports covariance and contravariance in **generic interfaces** and delegates and allows for implicit conversion of generic type parameters.

So it's telling that the framework alternate is present for netframework until 3.5 ... because once 4.0 comes, this method has contravariant generic parameters. As such, the c++/CLI formatter will only create signatures for methods that don't include variance.